cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -851,6 +851,17 @@ sub external_services {
         };
     }
 
+    if ( my $medium = $u->prop( 'medium' ) ) {
+	my $medium = LJ::eurl( $medium );
+	push @ret, {
+	    type => 'medium',
+	    text => LJ::ehtml( $medium ),
+	    url => "https://medium.com/\@$medium",
```

Comment:
I'm okay with either, too -- I don't  use Medium enough to say and just copied what the original person who asked for it to be added said.